test/e2e/network/firewall.go

```diff
@@ -167,7 +167,7 @@ var _ = SIGDescribe("Firewall rule", func() {
 		}
 
 		// Send requests from outside of the cluster because internal traffic is whitelisted
-		ginkgo.By("Accessing the external service ip from outside, all non-master nodes should be reached")
+		ginkgo.By("Accessing the external service ip from outside, all non-control plane nodes should be reached")
```

Comment:
nit - no need for dash here I think

```suggestion
		ginkgo.By("Accessing the external service ip from outside, all non control plane nodes should be reached")
```